src/components/Buttons/Button.tsx

```diff
@@ -14,6 +14,7 @@ export type ButtonProps = {
   onClick?: () => void;
   classNames?: string;
   id?: string;
+  className?: string;
```

Comment:
Please rework these prop names.

It is quite confusing to have two similar sounding names `classNames` and `className` which does different things. The original `classNames` is applies to the icon while the new `className` is applied to the entire button.

I suggest renaming these to `classNames` for the button and `iconClassNames` for the icon.